doc/contributing/adding-v8-fast-api.md

```diff
@@ -0,0 +1,119 @@
+# Adding V8 Fast API
+
+Node.js uses [v8](https://github.com/v8/v8) as the JavaScript engine.
```

Comment:
I think we always spell `V8` with an upper-case `V` (to avoid ambiguity with version numbers).